### PR TITLE
added a note to create a database before trying to create sql cache t…

### PR DESCRIPTION
**Pull request by aremo-ms**
Addressing #597 
Added a note to create a database prior to running SQL cache command

### CODE REVIEW

**Review by kalyankrishna1 — CHANGES_REQUESTED** [review]
Please run the latest codegen, update sample.json , test and then present back forr eview

**Comment by kalyankrishna1** [comment]
Look at and close this one too -> https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/136

File: 2-WebApp-graph-user/2-2-TokenCache/README.md [hunk 1]
```diff
@@ -148,13 +148,15 @@ Note: if you had used the automation to setup your application mentioned in [Ste
    > },
    > ```
 
-1. If you do not have an existing database and tables needed for token caching, you can execute the `dotnet sql-cache create` command to create the table for you. To do that, follow the steps below.
+1. If you do not have a tables needed for token caching, you can execute the `dotnet sql-cache create` command to create the table for you. To do that, follow the steps below.
```
Comment (jennyf19) [inline 2-WebApp-graph-user/2-2-TokenCache/README.md:151]:
```suggestion
1. If you do not have the tables needed for token caching, you can execute the `dotnet sql-cache create` command to create the table for you. To do that, follow the steps below.
```

**Review by jennyf19 — APPROVED** [review]
:shipit:

**Review by kalyankrishna1 — APPROVED** [review]

**Pull request closed without merging**